daisy_workflows/image_build/enterprise_linux/kickstart/rocky_linux_8_optimized_gcp_accelerator.cfg

```diff
@@ -0,0 +1,261 @@
+enabled=1
+gpgcheck=1
+repo_gpgcheck=0
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+       https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOM
+tee -a /etc/yum.repos.d/google-cloud.repo << EOM
+[google-cloud-sdk]
+name=Google Cloud SDK
+baseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el8-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=0
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+       https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOM
+
+# Rocky Linux Cloud Kernel repo.
+tee -a /etc/yum.repos.d/Rocky-CloudKernel.repo << EOM
+[cloud-kernel]
+name=Rocky Linux 8 - Cloud Kernel
+baseurl=https://dl.rockylinux.org/pub/sig/8/cloud/x86_64/cloud-kernel
+enabled=1
+gpgcheck=1
+gpgkey=https://dl.rockylinux.org/pub/sig/8/cloud/x86_64/cloud-kernel/RPM-GPG-KEY-Rocky-SIG-Cloud
+priority=-1
+EOM
+tee -a /etc/yum.repos.d/Rocky-CloudKernel.repo << EOM
+[cloud-kernel-source]
+name=Rocky Linux 8 - Cloud Kernel Source
+baseurl=https://dl.rockylinux.org/pub/sig/8/cloud/source/cloud-kernel
+enabled=0
+gpgcheck=1
+gpgkey=https://dl.rockylinux.org/pub/sig/8/cloud/x86_64/cloud-kernel/RPM-GPG-KEY-Rocky-SIG-Cloud
+priority=-1
+EOM
+# Be sure we don't get kernels from the BaseOS repo
+tee -a /etc/yum.repos.d/Rocky-BaseOS.repo << EOM
+exclude=kernel*
+EOM
+%end
+# Google Compute Engine kickstart config for Enterprise Linux 8.
+%onerror
+echo "Build Failed!" > /dev/ttyS0
+shutdown -h now
+%end
+
+%post --erroronfail
+set -x
+exec &> /dev/ttyS0
+# Delete the dummy user account.
+userdel -r gce
+
+# Import all RPM GPG keys.
+curl -o /etc/pki/rpm-gpg/google-rpm-package-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+curl -o /etc/pki/rpm-gpg/google-key.gpg https://packages.cloud.google.com/yum/doc/yum-key.gpg
+curl -o /etc/pki/rpm-gpg/RPM-GPG-KEY-Rocky-SIG-Cloud https://dl.rockylinux.org/pub/sig/8/cloud/x86_64/cloud-kernel/RPM-GPG-KEY-Rocky-SIG-Cloud
+rpm --import /etc/pki/rpm-gpg/*
+
+# Configure the network for GCE.
+# Given that GCE users typically control the firewall at the network API level,
+# we want to leave the standard Linux firewall setup enabled but all-open.
+firewall-offline-cmd --set-default-zone=trusted
+
+cat >>/etc/dhcp/dhclient.conf <<EOL
+# Set the dhclient retry interval to 10 seconds instead of 5 minutes.
+retry 10;
+EOL
+
+# Set google-compute-engine config for EL8.
+cat >>/etc/default/instance_configs.cfg.distro << EOL
+# Disable boto plugin setup.
+[InstanceSetup]
+set_boto_config = false
+EOL
+
+# Install GCE guest packages.
+dnf install -y google-compute-engine google-osconfig-agent gce-disk-expand
+
+# Install the Cloud SDK package.
+dnf install -y google-cloud-cli
+
+# Install Accelerator components: nvidia and mellanox drivers
+dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel8/x86_64/cuda-rhel8.repo
+dnf install -y gcc make kernel-devel kernel
```

Comment:
Drop lines 188-200 for now - we will just include the drivers when they are available from CIQ